### PR TITLE
feat(login): login base

### PR DESCRIPTION
**Pull request by ami**
（*’∀’人）♥

fixes https://github.com/robinjoseph08/pokedextracker.com/issues/14

### CODE REVIEW

File: app/views/login.html [hunk 1]
```diff
@@ -1 +1,17 @@
-<h1>Hi this is login page</h1>
+<form>
+  <div class="form-alert">
+    username not found
+  </div>
+  <div class="form-group">
+    <label for="username">Username</label>
+    <input id="username" type="text" required placeholder="ashketchum10">
```
Comment (robinjoseph08) [inline app/views/login.html]: are you able to limit the chars to 20 like reddit does for login and register? and limit password to 72 as well?

**Comment by robinjoseph08** [comment]
one thing but lmgtlmgtlmgtlmgt :+1:

File: app/views/login.html (hunk 1 above)
Comment (ami) [inline app/views/login.html]: should i add it for the register form too?

Comment (robinjoseph08) [inline app/views/login.html]: yup!

Comment (ami) [inline app/views/login.html]: actually it's kinda confusing to have a maxlength on creation b/c you might not be looking see that it didn't fit. nvm.

**Comment by ami** [comment]
updatedddd

**Pull request merged**